.goreleaser.yaml

```diff
@@ -1,16 +1,55 @@
 before:
   hooks:
     - go mod tidy
+
+env:
+  - GO111MODULE=on
+  - CGO_ENABLED=0
+
 builds:
-  - env:
-      - CGO_ENABLED=0
+  - binary: addlicense
     goos:
       - linux
       - windows
       - darwin
     goarch:
       - amd64
       - arm64
+dockers:
+  - image_templates:
+      - '{{ .Env.REGISTRY }}/{{ .Env.IMAGE_NAME }}:{{ .Tag }}-amd64'
```

Comment:
I don't love having these extra `*-{platform}` images getting tagged 😕 (as seen here: https://github.com/willnorris/addlicense/pkgs/container/addlicense)

I tried doing multi-architecture builds using the the regular docker actions to build the image, and it's definitely possible to do multi-architecture images without the intermediate tags (the `qemu` tag here: https://github.com/willnorris/imageproxy-test/pkgs/container/imageproxy-test).  I'm not sure if that can easily be done in goreleaser?